catenets/models/torch/utils/model_utils.py

```diff
@@ -81,10 +80,10 @@ def train_wrapper(
     **kwargs: Any,
 ) -> None:
     if hasattr(estimator, "train"):
-        log.debug(f"Train PyTorch network {estimator}")
+        print(f"Train PyTorch network {estimator}")

```

Comment:
What is the reason for this change from log to print?

Reply:
Hmm yeah I think I accidentally left a commit that shouldn't be there... I wonder can this commit be reverted but the other kept as it is...

Reply:
`git revert <commit hash>`

commit hash can be found in `git log`.